apps/src/applab/assetManagement/AssetRow.jsx

```diff
@@ -140,7 +146,7 @@ module.exports = React.createClass({
     }
 
     return (
-      <tr className="assetRow">
+      <tr className="assetRow" onDoubleClick={this.handleDoubleClick}>
```

Comment:
I think it's a native browser event: https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/ondblclick

It actually tripped me up for a minute, because I tried `onDblClick` first and that didn't work...